online-net-stock-alert/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Default values for online-net-stock-alert.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+suspend: true
+serverType: core-3-m-sata
+schedule: "* * * * *"
+
+# secretName:
+
+image:
+  repository: meyskens/online-net-stock-alert
+  tag: amd64-latest
```

Comment:
More and more I find that the whole idea about `latest` is evil. I'm sure somebody invented `pullPolicy: latest` in order to overcome that idea, but I'd rather prefer proper semantic versioning and make the world a better place. Please make the world a better place, @meyskens 😍

Reply:
I usually tag versions in docker, when pushing this script i did not as it wasn't that much script but for you i will add semver :wink: